sequencer/src/state_signature/relay_server.rs

```diff
@@ -0,0 +1,259 @@
+use crate::state_signature::StateSignatureScheme;
+
+use super::{LightClientState, StateSignatureRequestBody};
+use async_compatibility_layer::channel::OneShotReceiver;
+use async_std::sync::RwLock;
+use clap::Args;
+use ethers::types::U256;
+use futures::FutureExt;
+use hotshot_stake_table::vec_based::config::FieldType;
+use hotshot_types::light_client::{StateSignature, StateVerKey};
+use jf_primitives::signatures::SignatureScheme;
+use jf_utils::to_bytes;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::{BTreeSet, HashMap},
+    path::PathBuf,
+};
+use tide_disco::{
+    api::ApiError,
+    error::ServerError,
+    method::{ReadState, WriteState},
+    Api, App, Error as _, StatusCode,
+};
+use url::Url;
+
+/// The state signatures package is a light client state and its signatures collected
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct StateSignaturesPackage {
```

Comment:
Package sounds like a network package, what about `StateSignatureQuorum` or sth?

Reply:
Rename it to bundle in [1dde57c](https://github.com/EspressoSystems/espresso-sequencer/pull/1016/commits/1dde57c61e4683b1fde67bb5bde9e666a563311b)